### PR TITLE
Prepare 6 final default text for demo and local installs

### PR DESCRIPTION
**Pull request by stevepiercy**

### CODE REVIEW

File: src/plone/volto/default_homepage/default.py [hunk 1]
```diff
@@ -290,21 +290,25 @@
         },
     },
     "slate": {
-        "title": "Welcome to Volto!",
-        "description": "The React powered content management system",
+        "title": "Welcome to Plone 6!",
+        "description": "This site is Plone 6 with Volto for its frontend.",
```
Comment (sneridagh) [inline src/plone/volto/default_homepage/default.py:294]: I would remove the description block. It's not in the default pages anyways. What do you think?

Comment (stevepiercy) [reply src/plone/volto/default_homepage/default.py:294]: Done in 1c32bb49eb053948ea0ffb915679b30ccc746a2f

**Comment by stevepiercy** [comment]
@jenkins-plone-org please run jobs

**Comment by stevepiercy** [comment]
See https://github.com/plone/Products.CMFPlone/blob/6f1197eec1e0d94577885a1ffed8a85d09244e51/Products/CMFPlone/browser/templates/plone-frontpage.pt and its PR https://github.com/plone/Products.CMFPlone/pull/3700 for final version of default installation home page text for Classic UI. This should be adapted and translated for Volto.

@sneridagh note that I have a lot going on this week and will not be able to finish this until the weekend.

File: src/plone/volto/default_homepage/demo.py [hunk 2]
```diff
@@ -374,7 +373,7 @@
                                     {
                                         "children": [{"text": "The features of Plone"}],
                                         "data": {
-                                            "url": "https://plone.com/features.html"
+                                            "url": "https://plone.org/what-is-plone/plone/plone-features"
```
Comment (pbauer) [inline src/plone/volto/default_homepage/demo.py:376]: This needs to be changed to https://plone.org/why-plone/features

**Comment by mauritsvanrees** [comment]
We want to include this in final, right?
Checks are green. (Jenkins was not run, but seems not needed to me here.)
Can we merge and release?

**Review by fredvd — APPROVED** [review]
LGTM

**Comment by pbauer** [comment]
@jenkins-plone-org please run jobs

**Pull request merged**